api/builds/get.ts

```diff
@@ -35,18 +36,7 @@ export async function handler(
     });
   }
 
-  try {
-    new Bson.ObjectId(id);
-  } catch (_err: unknown) {
```

Comment:
Why is this bit removed?

Reply:
BSON ids are only used by mongo. Datastore doesnt have automatic ids, so instead we generate uuids. I guess I could do a check for it being either valid BSON or uuid if thats preferrable, but i dont see too much value in differentiating between invalid build id or just build not found just depending on the data format passed.

Reply:
Okay, as long as it doesn't break previous data it's fine